api.swagger.yaml

```diff
@@ -249,3 +490,1399 @@ paths:
+          $ref: '#/components/responses/NotFoundIDResponse'
+      description: Creates a new comment on the given event
+      tags:
+        - comment
+        - event
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CommentUpdate'
+  '/events/{id}/comments/{comment}':
+    parameters:
+      - schema:
+          type: string
+        name: id
+        in: path
+        required: true
+        description: The ID of the event on which this comment exists
+      - schema:
+          type: string
+        name: comment
+        in: path
+        required: true
+        description: The ID of the comment being manipulated
+    get:
+      summary: Retrieves a single comment on an event
+      tags:
+        - event
+        - comment
+      responses:
+        '200':
+          description: OK - The single comment response
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/SuccessfulAPIResponse'
+                  - type: object
+                    properties:
+                      result:
+                        $ref: '#/components/schemas/CommentResponse'
+        '401':
+          $ref: '#/components/responses/UnauthorizedResponse'
+        '403':
+          $ref: '#/components/responses/ForbiddenResponse'
+        '404':
+          $ref: '#/components/responses/NotFoundIDResponse'
+      operationId: get-events-id-comments-comment
+      description: Retrieves a single comment on this event
+    patch:
+      summary: Edits a given comment on a given event
+      operationId: patch-events-id-comments-comment
+      responses:
+        '200':
+          description: OK - the comment has been updated and returns the newly updated comment instance
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/SuccessfulAPIResponse'
+                  - type: object
+                    properties:
+                      result:
+                        $ref: '#/components/schemas/CommentResponse'
+        '400':
+          $ref: '#/components/responses/BadRequestResponse'
+        '401':
+          $ref: '#/components/responses/UnauthorizedResponse'
+        '403':
+          $ref: '#/components/responses/ForbiddenResponse'
+        '404':
+          $ref: '#/components/responses/NotFoundIDResponse'
+      description: Updates the given comment
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CommentUpdate'
+        description: 'All properties are optional, any value provided will be updated'
+      tags:
+        - event
+        - comment
+    delete:
+      summary: Deletes the comment from the event
+      operationId: delete-events-id-comments-comment
+      responses:
+        '200':
```

Comment:
According to https://docs.microsoft.com/en-us/azure/architecture/best-practices/api-design#delete-methods this should actually return `204`

Reply:
Yep

Reply:
Resolved via 800a3ba